swift/BUILD.bazel

```diff
@@ -22,108 +24,82 @@ filegroup(
     visibility = ["//swift:__subpackages__"],
 )
 
-pkg_files(
-    name = "dbscheme_files",
-    srcs = [
-        "ql/lib/swift.dbscheme.stats",
-        "//swift/extractor/trap:generated_dbscheme",
-    ],
+codeql_pkg_files(
+    name = "autobuilder-incompatible-os",
+    exes = ["//swift/tools/diagnostics:autobuilder-incompatible-os"],
 )
 
-pkg_files(
-    name = "manifest",
-    srcs = ["codeql-extractor.yml"],
+codeql_pkg_runfiles(
+    name = "autobuilder",
+    exes = ["//swift/swift-autobuilder"],
 )
 
 pkg_filegroup(
-    name = "extractor-pack-generic",
-    srcs = [
-        ":manifest",
-        "//swift/tools",
-    ] + select({
-        "@platforms//os:windows": [],
-        "//conditions:default": [
-            ":dbscheme_files",
-            "//swift/downgrades",
```

Comment:
I removed this distinction because the genric pack shouldn't really select on the platform, but this is what is causing the failure on the internal repo...

Reply:
turns out, windows _requires_ `--nobuild_python_zip` for the `//swift/extractor/trap:cppgen` rule to work :shrug: